Added Norbert Márkus to the AUTHORS file

hammera opened the pull request:
Hi Boys,

I have a suggestion that I have been thinking about for a long time:
Since February, Norbert has made very significant improvements to the file of the Liblouis math sub-table, in addition, in the case of the Hungarian language, he developed the Braille display of Greek letters all by himself the proper hungarian subtable (the subtable I already documented he with contributor), and brought the Nemeth Braille tables to the level of the Liblouisutdml project as much as possible. So, in Liblouisutdml level Norbert doed entire the nemeth.ctb and proper edit tables actualization related tasks (co-author), I provide only the technical support to Norbert have possibility only concentrate the table development.

If this is possible, I would like to suggest that you include Norbert in the Liblouis AUTHORS file among the table and test contributors, I think he deserves this honor.
Since Norbert is my colleague, and I know that he does not want to place his e-mail address in the AUTHORS file for the time being, I placed his name in the form of Norbert Márkus in my additional proposal in accordance with the English standard.
I don't know under what criteria someone can currently be included in the AUTHORS file, is there currently some set of rules that must be met in order to add someone to the list of official contributors.
I know for example if a people developing a full new table, usual landing right the AUTHORS file the proper place (with happened now for example the fillippino table author related), this is absolute right. But don't no what the situation with subtable related contributions level.

I hope you will accept my amendment proposal. If so, push this small change to the main branch.

Thank you in advance if you accept my suggestion positively, and sorry if my english not always right. :-):-)

Attila

egli commented:
I'm more than happy to add him, but I'd like an email address

hammera commented:
Chris, I consulted Norbert an e-mail.
Unfortunatelly he not enabled me to put the used e-mail adress in to the AUTHORS file, he enable only the name publication.
From a data protection point of view, I have to accept his decision, me not have other possibility.

Really not have possibility to we simple put he's name only the main AUTHORS file, similar with other oldest contributors with I showed my previous comment?

Attila

egli commented:
We would like to have a way to contact the authors, for whatever reason. Would it be possible to have the email somewhat obfuscated such as "name at gmail dot com"?

hammera commented:
Ok, in Liblouis the tables/nemethdefs.cti file Norbert defined with following form:
#  Expanded and modified by Norbert Márkus (hamilfonz at gmail dot com), Hungary, 2023.
In Liblouisutdml lbu_files/nemeth.ctb file are defined with following form:
#  Expanded and modified by Norbert Márkus (hamilfonz at gmail dot com), Hungary, 2023.
With nemeth_edit.ctb file I forgot final drop this form (when he sent me in august a rewritten version), perhaps I drop a PR this missing data related.
With nemeth_edit_en_ueb_with_switch_indicators.ctb Norbert are defined with following form:
#  Copyright (C) 2023 Norbert Márkus (hamilfonz at gmail dot com),
# Hungary
And the last nemeth edit table variant I defined he with following form:
#  Copyright (C) 2023 Norbert Márkus (hamilfonz at gmail dot com),
# Hungary

Personally I don't understand why he not would like adding he's e-mail address similar form with I showed you into the main authors file, He's a little eccentric sometimes, but he's a guy with absolutely good intentions and a great professional. :-):-)
So, table levels the contact data is awailable, except nemeth-edit.ctb file in Liblouisutdml, with I will be fixing a PR.

Attila
